inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/StreamSinkEntityMapper.java

```diff
@@ -117,6 +117,8 @@ List<String> selectExistsStreamId(@Param("groupId") String groupId, @Param("sink
      */
     List<SinkInfo> selectAllConfig(@Param("groupId") String groupId, @Param("idList") List<String> streamIdList);
 
+    List<StreamSinkEntity> selectAllStreamSinks();
```

Comment:
Suggest using `Cursor` to avoid the OOM.

Reply:
fixed, thx